### ISSUE TITLE
Error: UTF8 codes > 3 bytes are not currently supported. Or perhaps "-Gcharset=latin1" is needed?

### ISSUE THREAD

**Issue by jayhawk** [sensorflo/git-draw#1]
I got this error. Error: UTF8 codes > 3 bytes are not currently supported. Or perhaps "-Gcharset=latin1" is needed? Any suggestion on how to solve it? Thanks

**Comment by Mitmischer** [comment]
Try `git-draw -p > out.dot` and `dot -Gcharset=latin1 -Tpng out.dot > out.png`

**Comment by sensorflo** [comment]
You're right. But I don't have a clear idea how to solve it. Maybe one command line option to define the character encoding used for the content of all git objects. Internally that would be passed on to dot ala "-Gcharset=latin1" as you suggested.

**Comment by tiandalihe** [comment]
git-draw: command not found
why exist this problem?

**Comment by sensorflo** [comment]
Is the executable mode bit set? Being in the directory you installed git-draw do a ls like so

$ ls -la git-draw
-rwxr-xr-x 1 sensorflo sensorflo 11398 Feb 23 18:50 git-draw

The forth letter from the left must be an x like in the example above. You can set the executable bit like so

$ chmod u+x git-draw

When trying to execute, make sure the directory your git-draw script is in is in your PATH environment variable, or maybe better execute it like "./git-draw" instead just "git-draw".
